selftests/unit/test_data_structures.py

```diff
@@ -29,6 +29,16 @@ def test_geometric_mean(self):
         self.assertEqual(data_structures.geometric_mean(xrange(1, 180)),
                          67.1555819421869)
 
+    def test_compare_matrices(self):
+        """
+        Verify the correct value is produced when comparing matrices.
+        """
+        matrix1 = [[4, 51.7, 60], [6, 0, 0]]
```

Comment:
@ldoktor  Would you suggest this for replacing existing or to add as a second test ?
Can the `# Note` comment be included as well?